src/js/entities-service/entities/clinicians.js

```diff
@@ -42,12 +42,12 @@ const _Model = BaseModel.extend({
   addGroup(group) {
     const groups = this.getGroups();
     groups.add(group);
-    this.set('_groups', this.toRelation(groups, 'groups').data);
+    this.set('_groups', this.toRelation(groups, 'workspaces').data);
```

Comment:
The `toRelation`s in the entities and the the `type` change are the big changes.. not a huge deal.. largely FE ignores the types with the exception of the `includes` cache.